doc/panama_jextract.md

```diff
@@ -755,6 +755,137 @@ java -XstartOnFirstThread -Dforeign.restricted=permit --add-modules jdk.incubato
 
 ```
 
+## Using tensorflow (Mac OS)
+
+### getting libtensorflow
+
+* Download tensorflow library from
+
+    https://www.tensorflow.org/install/lang_c
+
+* extract the downloaded tar in a directory called LIBTENSORFLOW_HOME
+
+###  jextract c_api.h
+
+```sh
+
+jextract --source \
+  -I /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/ \
+  -t org.tensorflow \
+  -I ${LIBTENSORFLOW_HOME}/include \
+  -l ${LIBTENSORFLOW_HOME}/lib/libtensorflow.dylib \
+  ${LIBTENSORFLOW_HOME}/include/tensorflow/c/c_api.h
+
+javac --add-modules jdk.incubator.foreign org/tensorflow/*.java
+
+```
+
+### Python program that creates and saves model
```

Comment:
Might also want to mention here that the `tensorflow` package is needed to run the script.